plugin.php

```diff
@@ -168,6 +169,16 @@ function create_initial_rest_routes() {
 			$revisions_controller = new WP_REST_Revisions_Controller( $post_type->name );
 			$revisions_controller->register_routes();
 		}
+
+		foreach ( get_object_taxonomies( $post_type->name, 'objects' ) as $taxonomy ) {
+
+			if ( ! $taxonomy->public ) {
```

Comment:
Ah, I was looking at the taxonomies controller, I think need to update that too in that case.